tsconfig.json

```diff
@@ -17,5 +17,31 @@
     "filesGlob": [
         "node_modules/grafana-sdk-mocks/app/headers/common.d.ts",
         "src/**/*.ts"
+    ],
+    "files": [
```

Comment:
Won't this get matched by the wildcard in the filesGlob above? I assume it was auto-generated?